modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -206,4 +206,9 @@ The `hosts` parameter is a list of separate bare metal assets used to build the
 |
 | The MAC address of the NIC that the host uses for the `provisioning` network. Ironic retrieves the IP address using the `bootMACAddress` configuration setting. Then, it binds to the host.
 
+
+| `networkConfig`
+|
+| Set this optional parameter as an NMState YAML string denoted by the `\|` pipe character to configure the network interface of a host. See "(Optional) Configuring host network interfaces in the `install-config.yaml` file" for additional details.
```

Comment:
@johnwilkins 
As of 4.10.0-0.nightly-2022-01-21-231627 networkConfig no longer accepts string format (denoted by the pipe character)
it instead only accepts yaml format.
aside from removing the pipe character, behaviour is the same.